docs/documenting-schema.rst

```diff
@@ -0,0 +1,84 @@
+.. _documenting-schema:
+
+Documenting schema
+==================
+
+You can improve the experience of consuming your GraphQL API by documenting the types that it supports.
+
+Ariadne recommends writing documentation via the `description feature <https://facebook.github.io/graphql/June2018/#sec-Descriptions>`_ in GraphQL Schema Definition Language.  Keeping your documentation close to the code that supports it is a great way to ensure that it is always accurate and up-to-date.
+
+
+Example Description
+-------------------
+
+GraphQL descriptions are declared using a docstring format that feels very similar to Python's::
+
+    query = '''
+        """
+        Search results must always include a summary and a URL for the resource.
+        """
+        interface SearchResult {
+            "A brief summary of the search result."
+            summary: String!
+
+            "The URL for the resource the search result is describing."
+            url: String!
+        }
+    '''
+
+Note that GraphQL descriptions also support Markdown features (as specified in `CommonMark <https://commonmark.org/>`_)::
```

Comment:
I've didn't know this. 😯

How much of markdown spec is supported? We may want to confirm this with extra tests, or reword this part to something like "You can also use some of Markdown syntax in descriptions, such as text formatting or links.".